src/constants/icons.tsx

```diff
@@ -0,0 +1,16 @@
+import ETH from "public/assets/icons/currencies/eth.svg";
```

Comment:
minimal placeholders for poc, will fill out in a future pr once we have all the icon designs